### PR TITLE
Online notebooks

### PR DESCRIPTION
**Pull request by joelostblom**
Adding instruction for how to use Python without installing anything.

### CODE REVIEW

File: lessons/install-git-python-r.md [hunk 1]
```diff
@@ -12,21 +12,57 @@ Installation instructions for Git can be found on the
...
+   has many of the packages not in the default repositories.
+
+## Trying Python without installing
+
+There are a few online services that provide solutions to run Python and Jupyter
+Notebooks without installing anything. These often come with some limitations
+but is a simple way to just try things out:
+
+-   https://notebooks.ai/ Optional to create account (but necessary for saving
+    progress). Most up to date version of `seaborn`.
```
Comment (QuLogic) [inline lessons/install-git-python-r.md]: I don't know if we should mention this; it's liable to end up being out-of-date eventually.

File: lessons/install-git-python-r.md [hunk 2]
```diff
@@ -12,21 +12,57 @@ Installation instructions for Git can be found on the
...
+-   https://codeocean.com/ Account required. Sign up with institutional email
+    for extra storage and run time. Flexible but requires slight setup of
+    specifying which packages to install.
+-   https://colab.research.google.com/notebooks/welcome.ipynb Account required.
+    Google branded design.
+-   https://jupyter.org/try No account required. Not possible to save progress.
+    Lots of font-related warnings when plotting in our tests.
```
Comment (QuLogic) [inline lessons/install-git-python-r.md]: Same here; also, what warnings?

Comment (QuLogic) [reply lessons/install-git-python-r.md]: OK, never mind, I see the warnings with JupyterLab, but not the plain Python one.

**Comment by joelostblom** [comment]
I''m going to merge this because I want to link to the final version on the master branch.

**Pull request merged**